storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3071,10 +3071,9 @@ static int rocksdb_init_func(void *const p) {
                    MY_MUTEX_INIT_FAST);
 
 #if defined(HAVE_PSI_INTERFACE)
-  rdb_collation_exceptions =
-      new Regex_list_handler(key_rwlock_collation_exception_list);
+  rdb_collation_exceptions = new Regex(key_rwlock_collation_exception_list);
```

Comment:
Yes, their code style is roughly "whatever I feel like today". They did a major clang reformat but unfortunately did not place a .clang-format into their tree so we have some idea of what they style is and can 'enforce' it in our own changes. This is actually a good thing to ask them...